lib/msal-node/package.json

```diff
@@ -44,7 +45,22 @@
     "printWidth": 80,
     "semi": true,
     "singleQuote": true,
-    "trailingComma": "es5"
+    "trailingComma": "es5",
+    "endOfLine": "lf"
+  },
+  "jest": {
+    "verbose": true,
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "json",
+      "jsx",
+      "node"
+    ],
+    "testMatch": [
+      "<rootDir>/test/**/?(*.)(spec|test).(ts|js)?(x)"
```

Comment:
Looks like you need to pull in the branch that has the fix for this.

Reply:
Yes. I will into the base branch once I fold all the sub branches.